src/renderer/NumberInput.tsx

```diff
@@ -41,31 +74,85 @@ const NumberInput = ({
     }
 
     const newValue: number = parseInt(input, 10);
-    handleChange(newValue);
+    // Prevent user from typing numbers with more than 5 digits
+    if (newValue / 100000 > 1) {
```

Comment:
fails case when newValue = 100000 and for negative numbers
change to Math.abs(newValue) >= 100000